demisto_sdk/commands/content_graph/commands/update.py

```diff
@@ -49,6 +53,10 @@ def should_update_graph(
     )
 
 
+def get_packs_from_external_repo() -> list:
```

Comment:
1. this can work also for content repo, right? (if so - we can rename to `get_all_repo_pack_ids`)
2. let's move to `tools`

Reply:
1. Yes